### ISSUE TITLE
use it remotely ?

### ISSUE THREAD

**Issue by plopomi** [opentrack/opentrack#1094]
Hey, can I still use this program even if my phone and pc aren't in the same network ? (cloud gaming)

**Comment by sthalik** [comment]
Yes, as long as you can get UDP data through. There's no STUN/TURN support in opentrack though.
